include/config/config_goddard.h

```diff
@@ -18,3 +18,15 @@
  * Disables the demo that plays when idle on the start screen (has no effect if KEEP_MARIO_HEAD is disabled).
  */
 #define DISABLE_DEMO
```

Comment:
If there are no demos by default, this define should be removed tbh

Reply:
I reuse this to stub out demo code since I assume the majority of custom games will continue to not use this feature

Reply:
Would still prefer if it was removed or at least moved out of config into beginning of demo code file

Reply:
It's a bit too integral in multiple files (including the link script) to move out of a config header; What if I flip it to like an `ENABLE_DEMO_SYSTEM`, and have it commented by default instead?

Reply:
See how I have it now (on the correct account this time); now it's opt-in